build/build.js

```diff
@@ -29,7 +29,12 @@ async function build(opts) {
           __VERSION__: version,
           'process.env.SSR': false
         })
-      ])
+      ]),
+      onwarn: function (message) {
+        if (message.code === 'UNRESOLVED_IMPORT') {
+          throw new Error(`Could not resolve module ` + message.source)
```

Comment:
May be adding a suggestiong as well in the error message as "try running npm install"  will be helpful ?

Reply:
Message added in the following commit